catalogs/catalog-hadoop/build.gradle.kts

```diff
@@ -81,8 +81,15 @@ tasks {
 }
 
 tasks.test {
+  val skipUTs = project.hasProperty("skipTests")
+  if (skipUTs) {
+    // Only run integration tests
+    include("**/integration/**")
+  }
```

Comment:
Have you verified locally?

Reply:
Yes, I have verified the changes locally.